pkg/wakaama/Kconfig

```diff
@@ -27,6 +27,12 @@ config LWM2M_DEVICE_TTL
     help
         Lifetime of the device on the LwM2M server, expressed in seconds.
 
+config LWM2M_COAP_DEFAULT_BLOCK_SIZE
+    int "CoAP block size"
+    default 1024
+    help
+        Block transfer options support only power-of-two block sizes, from 2**4 (16) to 2**10 (1024) bytes.
```

Comment:
To constrain this you could either:
- expose only the exponent as a configuration, like so: https://github.com/RIOT-OS/RIOT/blob/faa10032ca7495f5c540eed1c84c49195718572c/drivers/at/Kconfig#L48-L57
- structure it as a `choice`